enhancements/node-tuning/pao-in-nto.md

```diff
@@ -102,11 +102,11 @@ that can be used to
 PAO code would be moved into the NTO repository as a separate controller. The Telco team would keep ownership as it has the necessary domain knowledge needed to implement the low latency tuning.
 
 The code currently uses Operator SDK libraries in some of the tests and this might need to be changed.
-Migrating the use of these libaries to [client-go](https://github.com/openshift/library-go)
+Migrating the use of these libaries to [library-go](https://github.com/openshift/library-go)
 would be one of the options.
 
 PAO also implements a [conversion webhook](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion)
-to migrate its custom resources during upgrade.
+to convert its custom resources during upgrade to versions that are supported by the openshift version targeted.
```

Comment:
my point is that this not during upgrade, but conversion are done in real-time on every request, even long after upgrade.

Reply:
@sttts  ack,  since this is a section about code in general it would make sense to drop the specific mention about upgrade for that paragraph then (following your comment)